### ISSUE TITLE
RPM created with distRpm does not install on RedHat 7.x

### ISSUE THREAD

**Issue by midmarch** [NCI-Agency/anet#3303]
Using
```
./gradle ./gradlew distRpm
cp distributions/anet-2.1.33~133~g529c3de10-2.x86_64.rpm /opt/ansible/anet/anet-2.1.33-133-g529c3de10-2.x86_64.rpm
```

```
[root@test-ansible-redhat76-01 ~]# yum localinstall /opt/ansible/anet/anet-2.1.33-133-g529c3de10-2.x86_64.rpm 
Loaded plugins: langpacks, product-id, search-disabled-repos
Examining /opt/ansible/anet/anet-2.1.33-133-g529c3de10-2.x86_64.rpm: anet-2.1.33~133~g529c3de10-2.x86_64
Marking /opt/ansible/anet/anet-2.1.33-133-g529c3de10-2.x86_64.rpm to be installed
Resolving Dependencies
--> Running transaction check
---> Package anet.x86_64 0:2.1.33~133~g529c3de10-2 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

====================================================================================================================================================================================================================
 Package                               Arch                                    Version                                                  Repository                                                             Size
====================================================================================================================================================================================================================
Installing:
 anet                                  x86_64                                  2.1.33~133~g529c3de10-2                                  /anet-2.1.33-133-g529c3de10-2.x86_64                                  115 M

Transaction Summary
====================================================================================================================================================================================================================
Install  1 Package

Total size: 115 M
Installed size: 115 M
Is this ok [y/d/N]: y
Downloading packages:
Running transaction check
Running transaction test


Transaction check error:
  file /etc/systemd from install of anet-0:2.1.33~133~g529c3de10-2.x86_64 conflicts with file from package systemd-219-62.el7_6.16.x86_64
  file /etc/systemd/system from install of anet-0:2.1.33~133~g529c3de10-2.x86_64 conflicts with file from package systemd-219-62.el7_6.16.x86_64

Error Summary
-------------
```

**Comment by midmarch** [comment]
Probably caused because the RPM tries to create folders like: `/etc/systemd` & `/etc/systemd`.
Source: https://serverfault.com/questions/775647/installing-rpm-package-to-systemd/775653

**Comment by midmarch** [comment]
Installation is fixed, remaining issue is: #3323
